### ISSUE TITLE
drive file deletion confirmation dialog is misleading

### ISSUE THREAD

**Issue by Johann150** [syuilo/Misskey#8370]
## 💡 Summary

The dialog text when deleting a drive file (`driveFileDeleteConfirm´) does not match the actual behaviour: When deleting a drive file, related notes do not get deleted. The file only gets removed as an attachment. This behaviour was changed in 07526ada453b10b329611393329780c1804b82d1, but the dialog text was not adjusted respectively.

As per the localisation README, only the ja-JP translation should be adjusted here and the remaining translations will be adjusted on Crowdin.

## 😁 Expected Behavior

Dialog accurately describes what will happen.

## 😕 Actual Behavior

Dialog says related notes will be deleted.

## 📌 Environment

Misskey version: since 12.96.0

**Comment by 4ioskd** [comment]
>When deleting a drive file, related notes do not get deleted. The file only gets removed as an attachment. 
https://github.com/misskey-dev/misskey/issues/8370#issue-1157002044

In misskey v12.108.1, same issue is occured. The note's sentence only exists.

**Comment by Johann150** [comment]
This is the line that has to be adjusted, preferrably by someone that speaks Japanese, so not me. https://github.com/misskey-dev/misskey/blob/ec3be7e4d362464a40722240e6210dfb45caed5f/locales/ja-JP.yml#L66

**Comment by 4ioskd** [comment]
This is just a proposal.

driveFileDeleteConfirm: "ファイル「{name}」を削除しますか？ただし、このファイルを添付したノートは消えません。"

**Comment by robflop** [comment]
I'd suggest incorporating both the notes with this attachment not getting deleted as well as the attachment vanishing from notes, as these are both non-obvious side effects imo.

Something like:

```yml
 driveFileDeleteConfirm: "ファイル「{name}」を削除しますか？このファイルを添付したノートは消えませんがノートのファイルは消えます。" 
```

**Comment by zyoshoka** [comment]
Fixed by #11465
